tensorflow_model_optimization/python/core/clustering/keras/cluster_wrapper.py

```diff
@@ -215,6 +215,10 @@ def update_clustered_weights_associations(self):
               pulling_indices, original_weight))
 
       if self.preserve_sparsity:
+        # Re-discover the sparsity masks to avoid zero-drifting
```

Comment:
This change is subject to #702 - to merge this one cleanly, please remove it from this PR.

Reply:
Will do. Thanks.